tests/testthat/test_write_fwf_blaise.R

```diff
@@ -183,3 +183,27 @@ test_that("dataframe can be written with small reals", {
 
   expect_silent(write_fwf_blaise(df, datafile, blafile, model_name = 'test'))
 })
+
+test_that("dataframe can be written with escaped strings", {
+  df = data.frame(
+    A = "0_escapes",
+    B = "1_\\escape",
+    C = "\\2esc\\"
+  )
+
+  datafile = tempfile(fileext = '.asc')
+  blafile = tempfile(fileext = '.asc')
```

Comment:
Beter om de extensie van de blafile hier .bla te maken. maakt verder niet uit voor de functionaliteit, maar maakt het makkelijker debuggen later (extensie is in de praktijk altijd bla voor een datamodel,